torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2170,8 +2170,8 @@ def sample_inputs_linalg_qr(op_info, device, dtype, requires_grad=False, **kwarg
     # TODO: add 0 to 'ns' and (0, ) to 'batches'
     # Currently tests fail most probably because of
     # https://github.com/pytorch/pytorch/issues/50576
-    batches = [(), (2, ), (1, 1)]
-    ns = [2, 5]
+    batches = [(), (0,), (2, ), (1, 1)]
```

Comment:
Updating sample inputs is cool, but I think we should extend the test_linalg.py tests for this, too?

Reply:
You're right, I'll update it.